gigareview/common/tests/unit/IngestTest.php

```diff
@@ -32,4 +32,90 @@ public function testTimestampBehaviourCreation()
         $ingest->save();
         $this->assertGreaterThan($ingest->created_at, $ingest->updated_at);
     }
+
+    public function testCreateIngestInstance()
+    {
+        $reportFileName = "Report-GIGA-em-manuscripts-latest-214-20220607004243.csv";
+        $scope = "manuscripts";
+        $reportType = 1;
+
+        $ingest = Ingest::createIngestInstance($reportFileName, $scope);
+        $this->assertNotNull($ingest);
+        $this->assertTrue(is_a($ingest, Ingest::class));
+        $this->assertEquals($reportFileName, $ingest->file_name);
+        $this->assertEquals($reportType, $ingest->report_type);
+        $this->assertTrue($ingest->isNewRecord);
+    }
+
+    public function testLogStatusAfterSave()
+    {
+        $reportFileName = "Report-GIGA-em-manuscripts-latest-214-20220607004243.csv";
+        $scope = "manuscripts";
+        $reportType = 1;
+        $fetchStatus = 3;
+        $remoteFileStatus = 1;
+        $parseStatus = 1;
+        $storeStatus = 1;
+
+        $updateStatusAfterSave = Ingest::logStatusAfterSave($reportFileName, $scope);
+        $this->assertTrue(true===$updateStatusAfterSave);
```

Comment:
Probably do not need `true===` because `$updateStatusAfterSave` will be a boolean value showing if `$ingest->save()` in the `logStatusAfterSave()` function succeeded or not.

Reply:
Good spot @pli888,

Yes, @kencho51 you either use:
```
$this->assertEquals(true, $updateStatusAfterSave);
```
or (more concisely):
```
$this->assertTrue($updateStatusAfterSave);
```